app/services/file_identifier_service.rb

```diff
@@ -14,7 +14,12 @@ def identify(filepath:)
     output, err, status = Open3.capture3('sf', '-json', filepath)
     raise Error, "Identifying #{filepath} returned #{status.exitstatus}: #{err}\n#{output}" unless status.success?
 
-    extract_file_types(output, filepath)
+    pronom_id, mimetype = extract_file_types(output, filepath)
+    return [pronom_id, mimetype] unless pronom_id == 'x-fmt/266' &&
```

Comment:
Doesn't the first clause (`x-fmt/266`) imply the second (`application/gzip`)?  Do we need both clauses?